examples/pytorch/model_zoo/chem/property_prediction/utils.py

```diff
@@ -30,9 +32,9 @@ def roc_auc_averaged_over_tasks(self):
             total_score += roc_auc_score(task_y_true, task_y_pred)
         return total_score / n_tasks
 
-
-class EarlyStopping:
+class EarlyStopping(object):
     def __init__(self, patience=10, filename="es_checkpoint.pth"):
+        assert not os.path.exists(filename), 'Filename {} is occupied.'.format(filename)
```

Comment:
This would make user have to change the filename every time.

Reply:
If you run this file for different combination of (model, dataset), some weird bug may occur.

Reply:
I didn't see why model would be loaded without being saved before at current using cases. 
It's a common scenario people using same filename for the early stopping model. If you really want to add this, you should delete the file after loading it.

Reply:
If you run the code on two different datasets, it's possible.